Sources/SpeziHealthKitBulkExport/BulkExportSession.swift

```diff
@@ -23,7 +23,6 @@ import SpeziLocalStorage
 /// ### Session State
```

Comment:
It might make sense to add some of the other properties about the different batches here / align the documentation in `BulkExportSessionProtocol` and here.

Reply:
i'll rework it to have the protocol be called `BulkExportSession`, rename the actual implementing class to `BulkExportSessionImpl` and make it private; that way we're not exposing the whole interface twice. since the protocol existentials can still carry the generic parameter, it'll work and behave exactly as if we were exposing the type directly.

Reply:
Sounds good!